### DISCUSSION TITLE
Can a module see which apps it is enabled for?

### DISCUSSION THREAD

**Discussion by Admicos** [LSPosed/LSPosed#1636] [Q&A]
I am making a module where I want the user to be able to toggle different behavior for different apps.

Is there a way a module can figure out which apps it is enabled for on the manager?

**Comment by mywalkb** [comment]
If you want control the scope of your module with the associated app, you must wait this TODO #9 
There is this PR #1593 which might be useful for your purpose if you want give root permission to your app, but now is in draft, so you must wait.
